.github/actions/build-cache/action.yml

```diff
@@ -6,5 +6,10 @@ runs:
     - name: Checkout repository
       uses: actions/checkout@v3
 
+    - name: Set up Node.js
+      uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+
```

Comment:
couldn't this be done here instead: https://github.com/KomodoPlatform/komodo-defi-framework/blob/main/.github/actions/deps-install/action.yml

that's more of the correct place for it.